src/twisted/python/_setup.py

```diff
@@ -348,3 +351,163 @@ def _checkCPython(sys=sys, platform=platform):
 
 
 _isCPython = _checkCPython()
+
+notPortedModules = [
```

Comment:
Do we really need to do this huge move in this branch?  It would be nice if we could avoid it on account of giant conflicts with every in-process py3 porting branch.